src/qt/walletmodel.cpp

```diff
@@ -561,9 +561,9 @@ bool WalletModel::getPubKey(const CKeyID &address, CPubKey& vchPubKeyOut) const
     return wallet->GetPubKey(address, vchPubKeyOut);
 }
 
-bool WalletModel::havePrivKey(const CKeyID &address) const
+isminetype WalletModel::isMine(const CTxDestination &dest) const
```

Comment:
`const CTxDestination& dest`.

Reply:
Where can I find guidelines about where to put the whitespace? I went with the existing layout. See how the line I replaced had `(const CKeyID &address)`?

Reply:
This is mentioned in the developer notes. Though, you could always just run clang format (also mentioned in the developer notes)